### PR TITLE
WebSocket.close: Silently ignore exceptions for close message

### PR DESCRIPTION
**Pull request by v1ne**
See commit message for details.

### CODE REVIEW

**Comment by v1ne** [comment]
@s-ludwig: Do you think this change is at the right place? This issue itself is quite annoying, even though I can only reproduce it stochasticly.

File: http/vibe/http/websockets.d [hunk 1]
```diff
@@ -699,12 +699,14 @@ final class WebSocket {
...
-					msg.write(cast(const ubyte[])reason);
-				}, FrameOpcode.close);
+			try {
+				send((scope msg) {
+					m_sentCloseFrame = true;
+					if (code != 0)
+						msg.write(std.bitmanip.nativeToBigEndian(code));
+						msg.write(cast(const ubyte[])reason);
+					}, FrameOpcode.close);
+			} catch (Exception) {}
```
Comment (s-ludwig) [inline http/vibe/http/websockets.d]: Would be good to add a `logDiagnostic("Failed to send active web socket close frame: %s", e.msg);` here.

Comment (v1ne) [reply http/vibe/http/websockets.d]: I'd consider this normal behaviour, but I added the log message, sure.
Also, I added a commit where I fix the close reply (the indentation has been misleading…).

**Comment by v1ne** [comment]
@s-ludwig: What do you think?

**Comment by s-ludwig** [comment]
Sorry, I lost track of this, but it looks good now!

**Pull request merged**